src/config/__tests__/load.ts

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+import load from '../load';
+
+jest.mock('../../lib/console');
+
+describe('should load config.yml', () => {
+  describe('invalid config', () => {
+    it('should return null on missing file', () => {
```

Comment:
IMO this should throw an error instead of returning null

Reply:
very true, we have a catch in config/index.ts instead